src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/legaladvisor/OrderCreator.java

```diff
@@ -96,26 +102,47 @@ public CallbackResponse prepopulateOrder(CallbackParams callbackParams) {
         logger.info("Order creator: pre populating order fields");
         CallbackRequest callbackRequest = callbackParams.getRequest();
 
-        Map<String, Object> data = new HashMap<>();
-        data.put(DIRECTION_LIST, ImmutableList.of(DOCUMENTS.name(), EYEWITNESS.name()));
-
         Claim claim = caseDetailsConverter.extractClaim(callbackRequest.getCaseDetails());
         CCDCase ccdCase = caseDetailsConverter.extractCCDCase(callbackRequest.getCaseDetails());
+
+        Map<String, Object> data = new HashMap<>();
+        data.put(DIRECTION_LIST, choseItem(ccdCase.getDirectionList(), ImmutableList.of(DOCUMENTS, EYEWITNESS)));
+
         addCourtData(claim, ccdCase, data);
 
         LocalDate deadline = legalOrderGenerationDeadlinesCalculator.calculateOrderGenerationDeadlines();
-        data.put(DOC_UPLOAD_DEADLINE, deadline);
-        data.put(EYEWITNESS_UPLOAD_DEADLINE, deadline);
-        data.put(DOC_UPLOAD_FOR_PARTY, BOTH.name());
-        data.put(EYEWITNESS_UPLOAD_FOR_PARTY, BOTH.name());
-        data.put(PAPER_DETERMINATION, NO.name());
+        data.put(DOC_UPLOAD_DEADLINE, choseItem(ccdCase.getDocUploadDeadline(), deadline));
+        data.put(EYEWITNESS_UPLOAD_DEADLINE, choseItem(ccdCase.getEyewitnessUploadDeadline(), deadline));
+        data.put(DOC_UPLOAD_FOR_PARTY, choseItem(ccdCase.getDocUploadForParty(), BOTH));
+        data.put(EYEWITNESS_UPLOAD_FOR_PARTY, choseItem(ccdCase.getEyewitnessUploadForParty(), BOTH));
+        data.put(PAPER_DETERMINATION, choseItem(ccdCase.getPaperDetermination(), NO));
+        data.put(ESTIMATED_HEARING_DURATION, choseItem(ccdCase.getEstimatedHearingDuration(), null));
```

Comment:
Suggestion - Do we need choseItem here as the default is going to be null anyway?